src/composables/useFlawModel.ts

```diff
@@ -140,7 +140,6 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), onSaveSuccess:
 
     try {
       await execute(...queue);
-      resetAffectChanges();
```

Comment:
Do we need to remove this? I think this is needed after flaw save. Because we don't need to show removed  affects after flaw save?

Reply:
ah i see. Just saw that you called this function on here. https://github.com/RedHatProductSecurity/osim/pull/243/files#diff-805151fe2771da356558582e49b1e66d6fc304a732ac683effea77dd71e7eba7R166